inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api_impl_defs.pxd

```diff
@@ -80,6 +116,25 @@ cdef extern from "<inference_engine.hpp>" namespace "InferenceEngine":
         const char *description
         apiVersion apiVersion
 
+    cpdef enum MeanVariant:
```

Comment:
It seems that C.MeanVariant, C.ResizeAlgorithm and C.ColorFormat are not used anymore and can be safely removed from here.

Reply:
They used as returned types in the PreProcessInfo. But their values can be removed.

Reply:
@nkogteva fixed